modules/users/client/views/profile/profile-view.client.view.html

```diff
@@ -155,6 +155,14 @@
                   Send a message
                 </a>
               </li>
+              <li ng-if="app.appSettings.referencesEnabled">
+                <a
+                  ui-sref="profile.experiences.new({username: profileCtrl.profile.username})"
+                >
+                  <i class="fa icon-message-alt"></i>
+                  Contact me via Phone
```

Comment:
Hard coded string? Shouldn't we use i18n keys?

Reply:
We should